extension-definition-specifications/incident-core/Incident Extension Suite.adoc

```diff
@@ -891,6 +896,10 @@ This list *MUST* not contain cycles.
 
 // tag::task-relationships[]
 
+When creating sequences of [stixtype]#<<task,tasks>># these *SHOULD NOT* be shared using relationship objects.
```

Comment:
Just to be sure it was intended, it leaves the option open to use relationships type as a SHOULD specified.

Reply:
During the last call, we need to check if the extension best practices allow to enforce a direction concerning relationships.